scripts/select_ladder_for_images.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import csv
+import os
+from typing import List, Optional
+
+import numpy as np
+from scipy.signal import find_peaks
+
+from DNAnet.data.data_models import Panel
+from DNAnet.data.data_models.hid_image import HIDImage, Ladder
+from DNAnet.data.utils import basepair_to_pixel
+from config_io import load_dataset
+
+OTHER_KITS = ("ppy23", "minifiler", "hdplex")  # other than PPF6C
+
+
+def get_ladder_paths(im: HIDImage) -> List[Optional[str]]:
+    """
+    Find all paths to ladder files in the same folder as the provided image.
+    """
+    return [im.path.parent / file for file in os.listdir(im.path.parent) if
+            ("ladder" in file.lower()
+             and not any([kit in file.lower() for kit in OTHER_KITS]))]
+
+
+def get_best_ladder_path(image: HIDImage, default_panel: Panel) -> Optional[str]:
```

Comment:
this algorithm only works if someone has already annotated the file (for which they needed a ladder). Write a TODO somewhere to get a working algorithm if we dont have an annotation yet. Or maybe we already have one?